README.md

```diff
@@ -1,26 +1,62 @@
 # ember-router-helpers
```

Comment:
This doesn't seem merged correctly?

Reply:
It's possible I changed too much / in the wrong spots. 

I've re-added the changes to the readme in a way that diffs easier